template/app_template.py

```diff
@@ -330,11 +312,11 @@ def run_simulation_impact_hours(event):
         param_with_tooltip(t.param.hatch_exchange_rate, tooltip='hatch_exchange_rate'),
         run_impact_hours
     ))
-    tmpl.add_panel('C', t.funding_pool_data_view)
+    tmpl.add_panel('C', t.outputs_overview_view)
     tmpl.add_panel('E', t.payout_view)
-    tmpl.add_panel('D', pn.Column(t.impact_hours_view, t.redeemable_plot))
+    tmpl.add_panel('D', pn.Column(t.impact_hours_plot, t.redeemable_plot))
```

Comment:
could you try to change the order of these two plots? IH chart is more readable and it would be nice to see it updating when running the simulation.